gn3/api/correlation.py

```diff
@@ -39,13 +57,17 @@ def compute_lit_corr(species=None, gene_id=None):
     are fetched from the database this is the only case where the db\
     might be needed for actual computing of the correlation results"""
 
-    database_instance = mock.Mock()
+    # database_instance = mock.Mock()
```

Comment:
*issue:* Remove the commented out code. Stale comments, given enough time, and enough people working on it, can be misleading when reading the code-base.